contrib/cirrus/setup.sh

```diff
@@ -25,5 +25,5 @@ set -x  # show what's happening
 curl --fail --location -o /tmp/netavark.zip "$NETAVARK_URL"
 mkdir -p /usr/libexec/podman
 cd /usr/libexec/podman
-unzip /tmp/netavark.zip
-chmod a+x /usr/libexec/podman/netavark
+#unzip -q /tmp/netavark.zip
+#chmod a+x /usr/libexec/podman/netavark
```

Comment:
Yeah, this is definitely going to break on arm64, there is no task on netavark to build non-x86.  IIRC, there's an arm cross-compile task over there, perhaps we could just arrange for it to publish a build artifact for aardvark CI to use?

Reply:
remind me, did we decide to get rid of the cross-compile task in favour of a separate aarch64 job like this one on netavark?